### ISSUE TITLE
Docs: Heroku free usage advice

### ISSUE THREAD

**Issue by thrize** [Steem-FOSSbot/steem-fossbot-voter#51]
After running two instances of the app on the same account for a few weeks, I can see that the Heroku free plan will only support **one instance at a rate of one bot run per hour, per month**.

This is very important information for new users so make this clear in the docs.

**Comment by williambanks** [comment]
You should also mention that Heroku requires a credit card and a lot of private personal details for signup.  Have you considered OpenShift?

**Comment by thrize** [comment]
I know it's a shame!

I did try to bring attention to that in the [installation doc](https://github.com/Steem-FOSSbot/steem-fossbot-voter/blob/master/docs/installation.md) here:

> Also be advised that you may have to verify your Heroku account to use the add-ons which provide auto bot scheduling and email sending. Verifying can involve a bank card but at time of writing does not require any transaction to be made.

If you think the wording should be update or the message differently highlighted, let me know.

I have used OpenShift before, years ago, and when I discovered Heroku I found the interface and integrations to be easier, and it's more "start-up-y" so I presume more attractive to people.

I would consider looking into deployment to OpenShift, feel free to raise it as feature request. My current priorities are on the local deployment, which might suit the more privacy minded better in any case, and I might even go as far as to make it a kind of stand alone desktop app using [Electron](http://electron.atom.io/). The issue is tracked at #9
